scripts/drush/benefits-category-make-pages.php

```diff
@@ -3,23 +3,28 @@
 
 use Drupal\Core\Url;
 use Drupal\node\Entity\Node;
+use Drush\Drush;
 
 // Create English and Spanish landing pages.
 // Useful after refreshing the database.
 $pageTypes = getTermMap('page_type');
 $benefitsSearchType = array_search('Benefits Category Search', $pageTypes);
 
 $englishNid = \Drupal::entityQuery('node')
-  ->condition('title', 'Find benefits by category')
+  ->condition('title', 'What government benefits go you qualify for?')
```

Comment:
"go" should be "do" here. I've been updating it manually, but I think it matters now. :-)

Reply:
I'm back! Sorry for the slow turnaround. I just realized what I meant to communicate was that I think the title the script is checking for should match the title it's creating, which I now realize is what you had in the first place. I'm pretty sure the script used to create pages with a "What government benefits go you qualify for" title.

Would it make more sense to query for the old page by page type and language?

Reply:
Yes, that makes infinitely more sense to search that way.